include/aca_on_demand_engine.h

```diff
@@ -17,22 +17,39 @@
 
 #define IPTOS_PREC_INTERNETCONTROL 0xc0
 #define DSCP_DEFAULT (IPTOS_PREC_INTERNETCONTROL >> 2)
-#define STDOUT_FILENO   1   /* Standard output.  */
+#define STDOUT_FILENO 1 /* Standard output.  */
 
 #include "common.pb.h"
 #include <openvswitch/ofp-errors.h>
 #include <openvswitch/ofp-packet.h>
 #include <string>
+#include <thread>
+#include "hashmap/HashMap.h"
+#include <grpcpp/grpcpp.h>
+#include <unordered_map>
 
 using namespace alcor::schema;
-
+using namespace std;
+// using namespace grpc;
+struct data_for_on_demand_call {
+  //     alcor::schema::OperationStatus on_demand_reply;
+  string uuid;
+  uint32_t in_port;
+  void *packet;
+  int packet_size;
+  alcor::schema::Protocol protocol;
+};
 // ACA on-demand engine implementation class
 namespace aca_on_demand_engine
 {
 class ACA_On_Demand_Engine {
   public:
+  std::thread *on_demand_reply_processing_thread;
+  grpc::CompletionQueue cq_;
```

Comment:
@zzxgzgz for an internal variable, we usually use '_' in front of a name, such as `_cq`

Reply:
fixed